src/core/Akka/IO/TcpManager.cs

```diff
@@ -54,6 +54,22 @@ internal sealed class TcpManager : ActorBase
     {
         private readonly TcpExt _tcp;
         
+        const string TcpListenerNamePrefix = "tcp-listener";
```

Comment:
Added some code here to force the `TcpManager` to give understandable, intent-driven names to its child actors. Should be self-explanatory.